_security/multi-tenancy/multi-tenancy-config.md

```diff
@@ -40,14 +44,14 @@ opensearch_security.multitenancy.tenants.preferred: ["Private", "Global"]
 opensearch_security.multitenancy.enable_filter: false
 ```
 
-Setting | Description
-:--- | :---
-`opensearch.requestHeadersAllowlist` | OpenSearch Dashboards requires that you add all HTTP headers to the allow list so that the headers pass to OpenSearch. Multi-tenancy uses a specific header, `securitytenant`, that must be present with the standard `Authorization` header. If the `securitytenant` header is not on the allow list, OpenSearch Dashboards starts with a red status.
-`opensearch_security.multitenancy.enabled` | Enables or disables multi-tenancy in OpenSearch Dashboards. Default is true.
-`opensearch_security.multitenancy.tenants.enable_global` | Enables or disables the global tenant. Default is true.
-`opensearch_security.multitenancy.tenants.enable_private` | Enables or disables the private tenant. Default is true.
-`opensearch_security.multitenancy.tenants.preferred` | Lets you change ordering in the **Tenants** tab of OpenSearch Dashboards. By default, the list starts with global and private (if enabled) and then proceeds alphabetically. You can add tenants here to move them to the top of the list.
-`opensearch_security.multitenancy.enable_filter` | If you have many tenants, you can add a search bar to the top of the list. Default is false.
+| Setting | Description |
+| :--- | :--- |
+| `opensearch.requestHeadersAllowlist` | OpenSearch Dashboards requires that you add all HTTP headers to the allow list so that the headers pass to OpenSearch. Multi-tenancy uses a specific header, `securitytenant`, that must be present with the standard `Authorization` header. If the `securitytenant` header is not on the allow list, OpenSearch Dashboards starts with a red status.
+| `opensearch_security.multitenancy.enabled` | Enables or disables multi-tenancy in OpenSearch Dashboards. Default is true. |
+| `opensearch_security.multitenancy.tenants.enable_global` | Enables or disables the global tenant. Default is true. |
+| `opensearch_security.multitenancy.tenants.enable_private` | Enables or disables the private tenant. Default is true. |
+| `opensearch_security.multitenancy.tenants.preferred` | Lets you change ordering in the **Tenants** tab of OpenSearch Dashboards. By default, the list starts with global and private (if enabled) and then proceeds alphabetically. You can add tenants here to move them to the top of the list. |
```

Comment:
Should "global" and "private" be capitalized or in quotes? They're capitalized in the first file.